docs/zh-CN/advanced/join-optimization/doris-join-optimization.md

```diff
@@ -0,0 +1,226 @@
+---
+{
+    "title": "Doris Join 优化原理",
+    "language": "zh-CN"
+}
+
+
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Doris Join 优化原理
+
+Doris 支持两种物理算子，一类是 **Hash Join**，另一类是 **Nest Loop Join**。
+
```

Comment:
doris doesn't really support  `Nest Loop Join`, it just uses an Nest-Loop-Join-like algorithm for non-equivalent joins, so I think this will cause some insurmountable problems for join optimization